keda/README.md

```diff
@@ -89,6 +89,11 @@ their default values.
 | `podIdentity.azureWorkload.enabled`                        | Specifies whether [Azure Workload Identity](https://azure.github.io/azure-workload-identity/) is to be enabled or not. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `false` |
 | `podIdentity.azureWorkload.tenantId`                       | Id Azure Active Directory Tenant to use for authentication with for Azure Workload Identity. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `` |
 | `podIdentity.azureWorkload.tokenExpiration`                | Duration in seconds to automatically expire tokens for the service account. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `3600` |
+| `podIdentity.irsa.audience`                                | Sets the token audience for IRSA. | `sts.amazonaws.com` |
+| `podIdentity.irsa.enabled`                                 | Specifies whether [AWS IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) is to be enabled or not. | `false` |
+| `podIdentity.irsa.roleArn`                                 | ARN of an IRSA IAM role with a web identity provider to use for authentication via STS. | `` |
+| `podIdentity.irsa.stsRegionalEndpoints`                    | Sets the use of an STS regional endpoint instead of global. Recommended to use regional endpoint in almost all cases. | `true` |
+| `podIdentity.irsa.tokenExpiration`                         | Duration in seconds to automatically expire tokens for the service account. | `86400` |
```

Comment:
Let's rename these to clarify it for end-users who are not familiar with AWS
```suggestion
| `podIdentity.awsIrsa.audience`                                | Sets the token audience for IRSA. | `sts.amazonaws.com` |
| `podIdentity.awsIrsa.enabled`                                 | Specifies whether [AWS IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) is to be enabled or not. | `false` |
| `podIdentity.awsIrsa.roleArn`                                 | ARN of an IRSA IAM role with a web identity provider to use for authentication via STS. | `` |
| `podIdentity.awsIrsa.stsRegionalEndpoints`                    | Sets the use of an STS regional endpoint instead of global. Recommended to use regional endpoint in almost all cases. | `true` |
| `podIdentity.awsIrsa.tokenExpiration`                         | Duration in seconds to automatically expire tokens for the service account. | `86400` |
```

Reply:
Maybe `podIdentity.aws.irsa.*` is even better in case AWS comes up with another provider (which we should have done for Azure as well)

Reply:
> Sounds like a good idea. I think this future proof option is also a better approach. So the agreement is that I update the PR to reflect the `podIdentity.aws.irsa.*` pattern?

Correct!

> I will keep the focus of the changes on AWS for his PR, but perhaps in another PR we can align Azure Workload Identity to follow the same pattern, but this would be a breaking change.

Yes, we can open an issue for that to track it but can't change it for now